articles/climate-risk-assessments/index.md

```diff
@@ -0,0 +1,157 @@
+
+To narrow scope, the workshop focused on a particular case study about the financial risk to one sector (electric utilities) impacted by one climatic hazard (extreme heat). This focus area was chosen for two reasons.
+
+First, electric utilities serve an important public function through the provision of essential services. Power outages pose serious health consequences, such as temperature-related illness or disruption of care for individuals who rely on electricity-dependent medical equipment.<Cite id='casey.2020' /> <Sidenote>According to the North American Electric Reliability Corporation’s (NERC) 2021 Long-Term Reliability Assessment, long duration weather events driven by climate change threaten reliability when demand for electricity is driven above forecasts and supplies are reduced, especially when extremely hot weather encompasses a wide area for an extended period of time.</Sidenote> Electric utilities have increasingly been impacted by extreme weather events, causing tragic loss of life and significant economic and health consequences. [For example](https://www.mckinsey.com/industries/electric-power-and-natural-gas/our-insights/why-and-how-utilities-should-start-to-manage-climate-change-risk), in 2017, Hurricane Harvey’s strong winds and record-breaking catastrophic flooding knocked down or damaged more than 6,200 distribution poles and 850 transmission structures across Houston, Texas. In California, an [investigation](https://www.theverge.com/2019/5/15/18626819/cal-fire-pacific-gas-and-electric-camp-fire-power-lines-cause) by the California Department of Forestry and Fire Protection (Cal Fire) found that the Camp Fire – California’s deadliest and most destructive wildfire on record – was caused by electrical transmission lines owned and operated by the Pacific Gas and Electric Company (PG&E).<Sidenote>PG&E [filed for Chapter 11 bankruptcy](https://www.washingtonpost.com/business/2019/01/29/pge-nations-biggest-utility-company-files-bankruptcy-after-california-wildfires/) following the deadly and record-breaking wildfire season.</Sidenote> More broadly, according to the [Fourth National Climate Assessment](https://nca2018.globalchange.gov/), utility infrastructure designed for historical climate conditions is more vulnerable to future weather extremes driven by climate change. Preparation for these changes, while crucial, also has associated costs. According to [recent estimates](https://www.mckinsey.com/industries/electric-power-and-natural-gas/our-insights/why-and-how-utilities-should-start-to-manage-climate-change-risk), preparing a typical Southeastern utility for the impacts of climate change will cost between $700 million and $1 billion.
```

Comment:
The combination of the `Cite` followed by an immediate `Sidenote` does not look great for sidenotes 8 and 9:
<img width="929" alt="CleanShot 2023-07-17 at 10 38 16@2x" src="https://github.com/carbonplan/research/assets/12436887/fa8c2656-441b-439f-bf6f-1c315c7f03b6">

This is something @orianac and I had talked about, but forgot to implement during the finalization step. Would it be possible to move the first `Sidenote` content into `references.json` to take advantage of the `Cite` grouping? This would look something like:

```
...electricity-dependent medical equipment.<Cite ids={['casey.2020', 'note.nerc']} />
```